Tools/TestWebKitAPI/Tests/WebCore/UserAgentQuirks.cpp

```diff
@@ -76,18 +76,26 @@ static void assertUserAgentForURLHasEmptyQuirk(const char* url)
     EXPECT_FALSE(uaString.isNull());
 }
 
-TEST(UserAgentTest, Quirks)
+static void assertUserAgentForURLHasNoQuirk(const char* url)
 {
     // A site with no quirks should return a null String.
-    String uaString = standardUserAgentForURL(URL("http://www.webkit.org/"_s));
+    String uaString = standardUserAgentForURL(URL(String::fromLatin1(url)));
     EXPECT_TRUE(uaString.isNull());
+}
+
+TEST(UserAgentTest, Quirks)
+{
+    assertUserAgentForURLHasNoQuirk("http://www.webkit.org/");
+
+    // We used to have a Chrome quirk for YouTube, added in: https://bugs.webkit.org/show_bug.cgi?id=253877
+    // Removed in: https://bugs.webkit.org/show_bug.cgi?id=289194
+    assertUserAgentForURLHasNoQuirk("http://youtube.com/");
```

Comment:
Sorry, I should have looked at this closer. I think you can safely delete this. We don't need to keep a history of previous quirks in this test.

```suggestion
```

I think this was actually the second or maybe even third YouTube user agent quirk. No doubt we'll need another quirk for this website in the future. :( But it's great that no quirks are required today.

Reply:
> We don't need to keep a history of previous quirks in this test.

But maybe we should? It makes sense that if you remove a quirk you would test for it not being applied aftwards.

Reply:
Shrug, it's harmless. You did the work, so you get to decide.